cmake/onnxruntime_providers.cmake

```diff
@@ -290,6 +290,7 @@ if (onnxruntime_USE_TENSORRT)
     set(CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -Wno-unused-parameter -Wno-missing-field-initializers")
   endif()
   set(CXX_VERSION_DEFINED TRUE)
+  list(APPEND CUDA_LIBRARIES cudart.lib cudadevrt.lib)
```

Comment:
See: https://cmake.org/cmake/help/v3.12/module/FindCUDA.html

The whole things in the page is deprecated.